### PR TITLE
Prioritized compound match 2

### PR DESCRIPTION
**Pull request by mikemol**
Resolves merge conflicts between #94 and #95

### CODE REVIEW

**Comment by alxwr** [comment]
Thanks!

File: openssh/files/sshd_config [hunk 1]
```diff
@@ -77,7 +89,7 @@
 # What ports, IPs and protocols we listen for
 {{ option('Port', 22) }}
 # Use these options to restrict which interfaces/protocols sshd will bind to
-{{ option('ListenAddress', ['::', '0.0.0.0']) }}
+{{ option('ListenAddress', ['::', '1.0.0.0']) }}
```
Comment (mikemol) [inline openssh/files/sshd_config]: Aiee! That can't be right!

Sorry, @alxwr, I missed that. I'm not really in a position to change it any more, but thought I'd give you a heads up.

**Comment by aboe76** [comment]
@mikemol just create another PR, if you have the time, or let me then I will create a PR with the fix

**Comment by alxwr** [comment]
@mikemol @aboe76 This code is obsolete. We dropped default values in #117, but even then someone changed it back to '0.0.0.0'.

```
% grep -r Listen openssh 
openssh/files/sshd_config:{{- option('ListenAddress') -}}
```
@mikemol But anyways: Thanks for the hint!

File: openssh/files/sshd_config (hunk 1 above)
Comment (alxwr) [reply openssh/files/sshd_config]: https://github.com/saltstack-formulas/openssh-formula/pull/96#issuecomment-453072904

**Pull request merged**